e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js

```diff
@@ -98,12 +104,12 @@ describe('Onboarding', () => {
             // # Check that 'Email Verified' text should be visible, email is pre-filled, and password field is focused, then login
             cy.findByText('Email Verified', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
             cy.get('#loginId').should('have.value', email);
-            cy.get('#loginPassword').should('be.focused').type(password);
```

Comment:
Focused on password field is as expected in manual check but it's not reliable to be checked in Cypress (maybe due to some re-render happening in a page) and so, I intentionally remove it in test.